src/main/java/frc/robot/stateMachine/StateMachine.java

```diff
@@ -0,0 +1,44 @@
+package frc.robot.stateMachine;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class StateMachine {
+    private String currentState;
+    private String initialState;
+    private HashMap<String, ArrayList<Edge>> edges;
+
+    public StateMachine(String init) {
+        initialState = init;
+        currentState = init;
+        edges = new HashMap<>();
+        edges.put(init, new ArrayList<Edge>());
+    }
+    public String getCurrentState() {
+        return currentState;
+    }
+    public void setInitialState(String state) {
```

Comment:
Maybe we don't actually need this function since you've got the intialization in the constructor